src/Provider/BuilderAliasProvider.php

```diff
@@ -88,7 +88,7 @@ private function getBuilder(string $bundleName, string $className): object
             }
 
             $builder = new $try();
-            if ($builder instanceof ContainerAwareInterface) {
+            if (interface_exists(ContainerAwareInterface::class) && $builder instanceof ContainerAwareInterface) {
```

Comment:
This change should not be necessary.

Reply:
> This change should not be necessary.

Are you sure? The interface was removed, so with some Symfony versions doesn't exist.

Reply:
Yes, but an `instanceof` on a non-existing class/interface is safe.

Reply:
Revert the change and see for yourself. The tests should pass. 🙂

Reply:
You were right. I reverted the change